.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ commands:
 
                 command: |
                   mv .riot .ddriot
-                  echo -p2.7,-p3.5,-p3.6,-p3.7,-p3.8,-p3.9 | tr ',' '\n' | circleci tests split | xargs ./scripts/ddtest riot -v run --exitfirst --pass-env -s '<< parameters.pattern >>'
+                  echo -p2.7,-p3.5,-p3.6,-p3.7,-p3.8,-p3.9,-p3.10 | tr ',' '\n' | circleci tests split | xargs ./scripts/ddtest riot -v run --exitfirst --pass-env -s '<< parameters.pattern >>'
```

Comment:
Crazy idea: add a Venv to the riotfile that generates this list of Python versions. This way it only needs to be kept up-to-date in one place.

Reply:
ouuu i like it. I'll give it a shot!